### PR TITLE
Trying fix for Windows

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,11 +11,12 @@ source:
   sha256: 3771a65699caee0f5b97cedada74a52b9ba4061bd8ad150d0115defb291c4581
 
 build:
-  number: 0
+  number: 1
   script:
     # The tarball shouldn't have egg-info--this one has paths in SOURCES.txt
     # that break the build on windows.
-    - rm -rf src/pyppmd.egg-info
+    - rd src/pyppmd.egg-info  # [win]
```
Comment (dopplershift) [inline recipe/meta.yaml]: It's complaining about a "bad switch" so try a Windows path:
```suggestion
    - rd src\pyppmd.egg-info  # [win]
```

**Comment by sarthakpati** [comment]
Still getting the same error on Windows 😞  [[ref](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=349005&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=1183ba29-a0b5-5324-8463-2a49ace9e213&l=543)]:

```
raise ValueError("path '%s' cannot be absolute" % pathname)
```

**Comment by dopplershift** [comment]
The bot is also complaining about doing this PR from the main repo. Can you fork and make a PR from your fork?

Also, [based on this](https://superuser.com/questions/179660/how-to-recursively-delete-directory-from-command-line-in-windows), try this command to delete:
```sh
rmdir src/pyppmd.egg-info /s /q
```

**Comment by sarthakpati** [comment]
Drafting this in favor or https://github.com/conda-forge/pyppmd-feedstock/pull/3

**Pull request closed without merging**